ui/desktop/electron-app/config/forge.config.js

```diff
@@ -1,12 +1,19 @@
 // Generated during build
 const config = require('./config.js');
 const { isMac } = require('../src/helpers/platform.js');
+const fs = require('fs');
+const path = require('path');
 
 console.log(`\n[forge-config] Release commit: ${config.releaseCommit}`);
 console.log(`[forge-config] Release version: ${config.releaseVersion}`);
 
-// MacOS signing identity
-if (isMac() && !process.env.BOUNDARY_DESKTOP_SIGNING_IDENTITY)
+// MacOS signing identity.
+// Ignore signing identity warning for debian builds run on MacOS
+if (
```

Comment:
Fine for now but going forward I think these conditionals are getting gnarly.  We need an elegant way to separate platform-specific concerns.

Reply:
Gotcha. I'll ponder over this after release.